.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: php
+
+php:
+  - '5.4'
+  - '5.5'
+  - '5.6'
+  - '7.0'
+
+before_script:
+  - git clone https://github.com/glpi-project/glpi -b 9.1 ../glpi && cd ../glpi
+  - mv ../formcreator plugins/formcreator
+  - composer install --no-dev
+  - php tools/cliinstall.php --db=glpi-test --user=travis --tests
+
+script:
+  - cd plugins/formcreator
+  - php tools/cliinstall.php --tests
```

Comment:
There is now a cliinstall.php in the plugin. It is designed to install the plugin itself for unit tests and may also be used to isntall the plugin from a CLI. The current directory for this call is the root directory of the plugin. 

That's why there a 2 calls to cliinstall.php.